src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -147,14 +144,8 @@
   <data name="TypeMethodMustReturnNotNullValue" xml:space="preserve">
     <value>The '{0}' method of type '{1}' cannot return a null value.</value>
   </data>
-  <data name="ActionSelector_GetCandidateActionsIsAmbiguous" xml:space="preserve">
-    <value>The supplied route values are ambiguous and can select multiple sets of actions.</value>
-  </data>
-  <data name="ArgumentPropertyUnexpectedType" xml:space="preserve">
-    <value>Property '{0}' is of type '{1}', but this method requires a value of type '{2}'.</value>
-  </data>
-  <data name="Common_ValueNotValidForProperty" xml:space="preserve">
-    <value>The value '{0}' is invalid.</value>
+  <data name="ModelBinding_NullValueNotValid" xml:space="preserve">
+    <value>A null value is invalid for '{0}'.</value>
```

Comment:
End users don't understand `null`. Maybe more like `"The {0} field cannot be empty."`?